### ISSUE TITLE
Assigning a texture to a resource of type "AnimatedTexture" indicates unsaved changes to the current scene.

### ISSUE THREAD

**Issue by Torguen** [godotengine/godot#68075]
### Godot version

v3.5.1.stable.official [6fed1ffa3]

### System information

w10 64

### Issue description

Godot indicates that there are unsaved changes to an empty scene currently open in the editor when you assign a texture to the "frame" property of a resource of type "AnimatedTexture".
That is, Godot indicates that the scene has unsaved changes even though the scene is not making use of that .tres file that you are configuring.

### Steps to reproduce

Create an empty scene.
Create a .tres AnimatedTexture file.
Assign a texture to the "frame" property of the .tres file you just created.
Notice that godot indicates that the empty scene has unsaved changes.

### Minimal reproduction project

...

**Comment by Calinou** [comment]
Does this occur with other resource types that are created independently from the currently edited scene? There's probably an issue for that already.

**Comment by Torguen** [comment]
Yes, this also happens with other resources like "NoiseTexture".
